hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/test/java/org/hypertrace/traceenricher/enrichment/enrichers/DefaultServiceEntityEnricherTest.java

```diff
@@ -218,7 +222,6 @@ public void test_case4_findServiceNameOfFirstAncestorThatIsNotAnExitSpanAndBelon
 
     StructuredTraceGraph graph = mock(StructuredTraceGraph.class);
     when(graph.getParentEvent(current)).thenReturn(parent2);
-    when(graph.getParentEvent(parent1)).thenReturn(parent1);
```

Comment:
Interesting

Reply:
Switching to use the mockito extension here (which uses strict stubs by default) highlighted a number of issues with the tests.